### ISSUE TITLE
HTTP 500: Unexpected error: <class 'AttributeError'> - 'str' object has no attribute 'get'

### ISSUE THREAD

**Issue by psegedy** [RedHatInsights/vmaas#330]
I get HTTP 500 error for `/cves, /errata, /repos` with `Unexpected error: <class 'AttributeError'> - 'str' object has no attribute 'get'`

for following string passed instead of JSON
```
"'key':'a quoted 'value' '"
```
```zsh
➜  vmaas_tests git:(missing_tests) ✗ curl -d @string3.json http://localhost:8080/api/v1/cves   
Unexpected error: <class 'AttributeError'> - 'str' object has no attribute 'get'%                         
➜  vmaas_tests git:(missing_tests) ✗ curl -d @string3.json http://localhost:8080/api/v1/repos
Unexpected error: <class 'AttributeError'> - 'str' object has no attribute 'get'%                         
➜  vmaas_tests git:(missing_tests) ✗ curl -d @string3.json http://localhost:8080/api/v1/errata
Unexpected error: <class 'AttributeError'> - 'str' object has no attribute 'get'%           
```
```
vmaas-webapp | Unexpected error: <class 'AttributeError'> - 'str' object has no attribute 'get'
vmaas-webapp | ERROR:tornado.access:500 POST /api/v1/errata (172.18.0.1) 1.29ms
```

`/updates` endpoint is able to handle it with HTTP error 400.
```zsh
➜  vmaas_tests git:(missing_tests) ✗ curl -d @string3.json http://localhost:8080/api/v1/updates
"'key':'a quoted 'value' '" is not of type 'object'% 
```
```
vmaas-webapp | ValidationError: "'key':'a quoted 'value' '" is not of type 'object'
vmaas-webapp | WARNING:tornado.access:400 POST /api/v1/updates (172.18.0.1) 6.07ms
```

IMHO all endpoints should treat it as malformed JSON and return HTTP 400 with ValidationError as `/updates` or maybe better with `Error: malformed input JSON.` like for the other malformed inputs.

**Comment by tkasparek** [comment]
Partially fixed by #329 as it brings behavior on par with /updates.
Te question is do we want to return some more meaningful error message instead of what's getting returned from validator?

**Comment by tkasparek** [comment]
As #339 made behavior of all APIs consistent (all of them are returning 400 ValidationError), closing this issue.
